src/renderer/App.tsx

```diff
@@ -102,14 +116,22 @@ export default function App() {
   // game (back button or home button), team users should be navigated to their home screen too. The 'quitGame' message is send to them for
   // this purpose.
   router.subscribe((x) => {
+    // case when you exit game-host screen without finishing game
     if (
       prevLocationRoute &&
       prevLocationRoute === RouteDefinitions.GameHost.route &&
       x.location.pathname !== RouteDefinitions.GameSettings &&
+      x.location.pathname !== RouteDefinitions.GameFinished &&
```

Comment:
should this IF be extended every time we define new routes?

Reply:
The logic inside router.subscribe is used to detect, if the running game was closed. Imagine team screens. They first see Welcome Team page. And as soon as host starts a game, they see "wait for next question" screen. However, if host closes the game (by navigating back, or clicking home button), or the game finishes, we want team users to redirect back to their home screen (welcome team). For that we are using this send message 'quitGame'. We are storing previous screen on host, and comparing it with current screen, by this we know if he cancelled or finished the game. So this logic needs to be adapted only if we add new screen to the running game.